backend/apps/progress/management/commands/refresh_leaderboard.py

```
@@ -0,0 +1,25 @@
+    def handle(self, *args, **options):
+        self.stdout.write("Refreshing progress_leaderboard_mv...")
+        try:
+            with connection.cursor() as cursor:
+                # CONCURRENTLY requires a unique index on the MV which we created.
+                cursor.execute(
+                    "REFRESH MATERIALIZED VIEW CONCURRENTLY progress_leaderboard_mv;"
+                )
+            self.stdout.write(
+                self.style.SUCCESS("Successfully refreshed progress_leaderboard_mv")
+            )
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)